pom.xml

```diff
@@ -74,22 +74,22 @@
       
       <repository>
          <id>eclipse.platform.4.10</id>                                  <layout>p2</layout>
-         <url>jar:file:repositories/org.eclipse.platform-4.10.zip!/</url>
```

Comment:
The following 4 modified lines are within a comment and should not be modified.

The buildprocess for 19.10 worked with eclipse 4.13 repositoy